### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by davidanthoff**

### CODE REVIEW

File: src/performance.jl [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 include("mimi-page/src/montecarlo.jl")
 
-df = DataFrame([String,Bool,Int,Float64],[:model,:multithreading,:runs,:elapsed],0)
+df = DataFrame([String,String,Int,Float64],[:model,:multithreading,:runs,:elapsed],0)
 
 for i in (1_000, 10_000)
     timing = do_monte_carlo_runs(i)
 
-    push!(df, ("Mimi-PAGE", false, i, timing))
+    push!(df, ("Mimi-PAGE", "false", i, timing))
```
Comment (jrising) [inline src/performance.jl]: Is this for working with the CSV package?

Comment (davidanthoff) [reply src/performance.jl]: Yes, for some odd reason it doesn't detect ``Bool`` columns properly...

Comment (jrising) [reply src/performance.jl]: I've run into a lot of problems with the CSV package.  I wish they had offered a package with the same functionality as was in DataFrames.

Comment (davidanthoff) [reply src/performance.jl]: Which one are you trying? There are two: CSV.jl and CSVFiles.jl. The latter is maintained by me and what we are using here. The ``Bool`` thing clearly needs to be fixed...

Comment (jrising) [reply src/performance.jl]: Oh, that's why you're using the load and save functions.  Sorry, I was thinking of the CSV.jl package, which I've temporarily given up on.  I should try FileIO.

**Pull request merged**